docs/index.rst

```diff
@@ -207,6 +207,45 @@ with 3 items while processing a request for `/social/list`.
     ...     assert current_menu.submenu('account.list').active
     ...     current_menu.children
 
+
+Flask-Classy
+===
+
+Flask-Classy is a library commonly used in Flask development and gives 
+additional structure to apps which already make use of blueprints. 
+
+Using Flask-Menu with Flask-Classy is rather simple:
+
+.. code-block:: python
+
+    from flask.ext.classy import FlaskView
```

Comment:
for Flask 1.0 we should rather use `flask_<extension>` hence `from flask_classy import ...` and `import flask_menu as menu`